fuel-tx/src/transaction.rs

```diff
@@ -414,30 +414,23 @@ pub trait Executable: field::Inputs + field::Outputs + field::Witnesses {
         self.inputs_mut().push(input);
     }
 
-    /// Prepare the transaction for VM initialization for script execution
-    ///
-    /// note: Fields dependent on storage/state such as balance and state roots, or tx
-    /// pointers, should already set by the client beforehand.
-    fn prepare_init_script(&mut self) -> &mut Self {
-        self.outputs_mut()
-            .iter_mut()
-            .for_each(|o| o.prepare_init_script());
-
-        self
-    }
-
-    /// Prepare the transaction for VM initialization for predicate verification
-    fn prepare_init_predicate(&mut self) -> &mut Self {
-        self.inputs_mut()
-            .iter_mut()
-            .for_each(|i| i.prepare_init_predicate());
-
-        self.outputs_mut()
-            .iter_mut()
-            .for_each(|o| o.prepare_init_predicate());
-
-        self
-    }
+    // /// Prepare the transaction for VM initialization for script or predicate execution
```

Comment:
was this meant to be left commented out?

Reply:
No. Removed in 41b2e19f.